packages/synapse-react-client/src/components/Authentication/TwoFactorAuthSettingsPanel.stories.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react'
 import { Meta, StoryObj } from '@storybook/react'
-import { displayToast } from '../ToastMessage/ToastMessage'
-import TwoFactorAuthSettingsPanel from './TwoFactorAuthSettingsPanel'
+import { displayToast } from '../ToastMessage'
+import TwoFactorAuthSettingsPanel, {
+  TwoFactorAuthSettingsPanelProps,
+} from './TwoFactorAuthSettingsPanel'
 import { Paper } from '@mui/material'
 
-const meta = {
+const meta: Meta<TwoFactorAuthSettingsPanelProps> = {
```

Comment:
Fix a handful of TypeScript errors that get logged to the console when bundling.